shared/templates/create_audit_rules_usergroup_modification.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python2
+
+#
+# create_audit_rules_usergroup_modification.py
+#        generate template-based checks for user/group modification rules
+
+
+from template_common import FilesGenerator, UnknownTargetError
+
+import os
+import re
+
+class AuditRulesUserGroupModificationGenerator(FilesGenerator):
+    def generate(self, target, args):
+        path = args[0]
+        name = re.sub('[-\./]', '_', os.path.basename(path))
+        if target == "oval":
+            self.file_from_template(
+                "./template_OVAL_audit_rules_usergroup_modification",
+                {
+                    "%NAME%":	name,
+                    "%PATH%":	re.sub("/", "\/", path)
```

Comment:
We shouldn't rely on unrecognized escape sequences. Is there any reason why we shouldn't go with the proper `\\/` here?